### PR TITLE
[RHPAM-4026] - [OpenShift] Users and Groups are not displayed in Admi…

### PR DESCRIPTION
**Pull request by spolti**
…n perspective
See: https://issues.redhat.com/browse/RHPAM-4026

Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by msmagnanijr — DISMISSED** [review]

**Review by desmax74 — DISMISSED** [review]

**Review by desmax74 — APPROVED** [review]

**Review by msmagnanijr — APPROVED** [review]

**Review by amadhusu — APPROVED** [review]

File: businesscentral/image.yaml [hunk 1]
```diff
@@ -118,6 +121,9 @@ envs:
   - name: "KIE_DASHBUILDER_RUNTIME_LOCATION"
     example: "http://dashbuilder:8080"
     description: "Enable integration between Business Central and Dashbuilder."
+  - name: "KIE_GIT_CONFIG_PATH"
+    example: "/opt/my_git_config.json"
+    description: "SSO configuration for ssh authentication on Decision Central. Not required, if not set will configure one based on the given SSO properties and will store under /opt/eap/kie_git_config.json"
```
Comment (jakubschwan) [inline businesscentral/image.yaml:126]:
```suggestion
    description: "SSO configuration for ssh authentication on Business Central. Not required, if not set will configure one based on the given SSO properties and will store under /opt/eap/kie_git_config.json"
```

File: templates/rhpam713-authoring-ha.yaml [hunk 2]
```diff
@@ -421,15 +421,6 @@ parameters:
     name: AUTH_LDAP_BIND_CREDENTIAL
     example: "Password"
     required: false
-  - displayName: LDAP Flag Login Module
-    name: AUTH_LDAP_LOGIN_MODULE
-    description: "A flag to set login module to optional. The default value is required"
-    example: "optional"
-    required: false
```
Comment (jakubschwan) [inline templates/rhpam713-authoring-ha.yaml:428]: @spolti is possible to keep this flag in templates/images and have fallback method for LDAP login module? We should keep this functionality in images if it is possible

**Comment by mareknovotny** [comment]
could you please review this PR as top priority? we need to merge this today for our release.  @jakubschwan @msmagnanijr @vaibhavjainwiz

**Review by jakubschwan — APPROVED** [review]
lgtm

**Review by desmax74 — APPROVED** [review]

**Review by desmax74 — APPROVED** [review]

**Review by amadhusu — APPROVED** [review]

**Pull request merged**